spring-cloud-deployer-local/src/main/java/org/springframework/cloud/deployer/spi/local/LocalDeployerProperties.java

```diff
@@ -170,6 +170,16 @@ public String toString() {
 
 	private boolean inheritLogging;
 
+	private String dockerNetwork = "bridge";
+
+	public String getDockerNetwork() {
+		return dockerNetwork;
+	}
+
+	public void setDockerNetwork(String dockerNetwork) {
```

Comment:
Would it be better if we'd start to group these settings, like `spring.cloud.deployer.local.docker.network`. Might look more clean especially if we add more settings.

Reply:
makes sense. Just changed it into docker.network and added some related tests.